tasks/parse-version-windows.yml

```diff
@@ -3,13 +3,15 @@
 - name: Get Windows Agent version
   win_shell: |
     $product_name = "Datadog Agent"
-    $query = "Select Name,IdentifyingNumber,InstallDate,InstallLocation,ProductID,Version FROM Win32_Product where Name like '$product_name%'"
-    $installs = Get-WmiObject -query $query
+    $version=Get-ItemProperty HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\* |
```

Comment:
compared to the initial proposal the `HKLM:\SOFTWARE\WOW6432Node...` part is missing.
I don't know what it covers on top but it could be interesting to include as well, wdyt?

Reply:
I do not think it is required, I mostly took the code from the second comment, that seems working correctly:
https://github.com/DataDog/ansible-datadog/issues/526#issuecomment-1850422352

Reply:
FYI: Searching in both (with the one 'Wow6432Node') allows for broader support (namely x86 apps on x64). 

More precisely: "Wow6432 means you’re running a 64-bit version of Windows and allows to manage 32-bit applications that runs on a 64-bit version of Windows. When the 32-bit app performs a registry query on HKEY_LOCAL_MACHINE\SOFTWARE\<company>\<product>, Windows automatically redirects the query to HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\<company>\<product>".

Since I carry such commands in many cases (software), it leaked in my snippet/example. I leave this up to you to determine if that can happen (e.g. installing an x86 version of Datadog Agent on a  64-bit version of Windows).

No matter what, thanks for taking care of this issue. It's very appreciated.

Reply:
That's interesting, thank you for the explanation! Since we do not provide a 32bits version of the Agent I guess that we do not need to search in `Wow6432Node`

Reply:
I guess this will eventually land/release on https://github.com/ansible-collections/Datadog as well? It is where we fetch it.

Reply:
Sorry for the delay. The new version of the role and the collection was finally released. 
On Ansible Galaxy: 
 - role: https://galaxy.ansible.com/ui/standalone/namespaces/1584/
 - collection: https://galaxy.ansible.com/ui/repo/published/datadog/dd/